src/main/resources/flows-config.yaml

```diff
@@ -15,6 +15,7 @@ flow:
     nextScreens:
       - name: redirectToMDThink
   selectHelpNeeded:
+    beforeSaveAction: PreCheckPrograms
```

Comment:
@bseeger - any strong sentiment between adding this as a beforeSave action or a before display action on the next page?

Reply:
I think it was already checking them based on the value of the help needed...?